cmd/vic-init/ops.go

```diff
@@ -69,6 +70,9 @@ func (t *operations) HandleSessionExit(config *tether.ExecutorConfig, session *t
 			return
 		}
 
+		// incredibly basic throttle
+		time.Sleep(3 * time.Second)
```

Comment:
3 seconds seems pretty long for throttle. Something sub-second will no do the job here?